integrations/intercom/lib/index.js

```diff
@@ -23,6 +23,7 @@ var Intercom = (module.exports = integration('Intercom')
   .option('activator', '#IntercomDefaultWidget')
   .option('appId', '')
   .option('richLinkProperties', [])
+  .option('apiBase', '')
```

Comment:
So Intercom isn't planning to make this required right? Wondering if we should default to the current value (US?) - what does the option look like in the app if you have 3 choices but none selected?

Reply:
Actually originally added the default (`https://api-iam.intercom.io`) here, but then thought that's not 100% backwards compatible, because it forces users to use `https://api-iam.intercom.io`, while the old behavior doesn't specify the API host explicitly. Although I'm not 100% sure if defaulting to `https://api-iam.intercom.io` would cause any issues, but if somehow some users override the API host, and then we override it again it will be bad.

Reply:
Gotcha, makes sense!